File: src/test/bench/fib/fib.rb
start = Time.now

def fib(n)
  if n < 2 then
    n
  else
    fib(n - 1) + fib(n - 2)
  end
end

res = fib(30)
puts "time: " + ((Time.now - start) * 1000).to_s
puts res

File: src/test/bench/fib/fib.js
const start = Date.now()

function fib(n) {
    if (n < 2) {
        return n
    }
    return fib(n - 1) + fib(n - 2)
}

const res = fib(30)
console.log(`time: ${Date.now() - start}`)
console.log(res)
